tests/AttributeExtensionTest.php

```diff
@@ -0,0 +1,124 @@
+<?php
+declare(strict_types=1);
+
+namespace Zalas\PHPUnit\Globals\Tests;
+
+use PHPUnit\Framework\TestCase;
+use PHPUnit\Runner\AfterTestHook;
+use PHPUnit\Runner\BeforeTestHook;
+use Zalas\PHPUnit\Globals\AnnotationExtension;
+use Zalas\PHPUnit\Globals\Attribute\Env;
+use Zalas\PHPUnit\Globals\Attribute\Putenv;
+use Zalas\PHPUnit\Globals\Attribute\Server;
+
+#[Env('APP_ENV', 'test')]
+#[Server('APP_DEBUG', '0')]
+#[Putenv('APP_HOST', 'localhost')]
+class AttributeExtensionTest extends TestCase
+{
+    public function test_it_is_a_test_hook()
+    {
+        $this->assertInstanceOf(BeforeTestHook::class, new AnnotationExtension());
+        $this->assertInstanceOf(AfterTestHook::class, new AnnotationExtension());
+    }
+
+    #[Env('APP_ENV', 'test_foo')]
+    #[Server('APP_DEBUG', '1')]
+    #[Putenv('APP_HOST', 'dev')]
+    public function test_it_reads_global_variables_from_method_annotations()
```

Comment:
Method names in this test case should use "attributes" instead of "annotations".